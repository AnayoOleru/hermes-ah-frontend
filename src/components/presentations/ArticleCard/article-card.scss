@import '../../../assets/stylesheets/index.scss';
.article-card-item{
    .author-name {
        color: black;
        font-weight: 600
    }
    
    .category {
        text-transform: uppercase;
        margin: 0;
        color: #0E71EB;
    } 

    .description:hover {
        color: #4183c4 !important;
        cursor: pointer;
    }

    a.header {
        text-transform: capitalize;
    }

}
.ui.items>.item>.image>img {
    height: 169px;
}
.twelve.wide.computer.sixteen.wide.mobile.column{
    height: 69vh;
    overflow-y:auto;
}

.twelve.wide.computer.sixteen.wide.mobile.column::-webkit-scrollbar-track
{
	background-color: #ffffff;
}

.twelve.wide.computer.sixteen.wide.mobile.column::-webkit-scrollbar-thumb
{
	background-color: #ffffff;
}
.twelve.wide.computer.sixteen.wide.mobile.column:hover::-webkit-scrollbar-thumb
{
	border-radius: 10px;
	-webkit-box-shadow: inset 0 0 6px rgba(255, 255, 255, 0.3);
	background-color: #8f8f8f;
}
