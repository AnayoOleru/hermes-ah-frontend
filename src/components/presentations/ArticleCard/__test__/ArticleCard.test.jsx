import React from 'react';
import { shallow } from 'enzyme';
import ArticleCard from '../Article';
import mock from '../../../../utils/testMocks';

const { article } = mock;

describe('<ArticleCard />', () => {
  const props = {
    key: article.id,
    category: article.category,
    title: article.title,
    author: article.author,
    date: article.date,
    read: article.read,
    paragraph: article.paragraph,
    image: article.image,
  };
  it('renders article', () => {
    const wrapper = shallow(<ArticleCard {...props} />);
    expect(wrapper.find('div'));
    expect(wrapper.toJSON).toMatchSnapshot();
  });
});
