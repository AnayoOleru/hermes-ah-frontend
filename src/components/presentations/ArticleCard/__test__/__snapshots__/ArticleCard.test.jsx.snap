// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ArticleCard /> renders article 1`] = `undefined`;
